llmfoundry/data/text_data.py

```diff
@@ -232,6 +232,19 @@ def get_sequence_id_from_batch(
         return torch.cat([left_zeros, cumulative_sep[:, :-1]], dim=1)
 
 
+def build_streams(dataloader_cfg: DictConfig):
```

Comment:
should this just take in the dataset config instead of the dataloader cfg?

Reply:
ah yeah, nice catch.